### PR TITLE
Add context and goals of this repo in the README

### PR DESCRIPTION
**Pull request by jorisvandenbossche**
I mainly want to ensure with this README that when pointing people towards this repo (eg when trying to engage other communities, emailing GDAL etc), it is clear what the reason is for this repo and its goals.

cc @kylebarron in case you can take a look to see if you find this clear / covers your expectations / what else you would add?

### CODE REVIEW

**Comment by jorisvandenbossche** [comment]
also cc @brendan-ward

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+with feature geometries and additional columns with feature attributes. 
+
+Defining a standard way to store geospatial data in the Arrow memory layout
+can help interoperability between different tools and enable fast data
+transfer:
+
+- Ensure that different tools can read Parquet or Feather files with geospatial
+  data and correctly interpret the geometry information. Concrete example: we
+  want that the R `sf` package can read a parquet file produced by the Python
+  `geopandas` package and vice versa.
```
Comment (kylebarron) [inline README.md:25]: A simple description of Feather might be helpful.

**Comment by brendan-ward** [comment]
@jorisvandenbossche this looks good - thanks for working on this!

File: README.md (hunk 1 above)
Comment (jorisvandenbossche) [reply README.md:26]: Added a short sentence about it in the paragrapgh about Apache Arrow.

**Comment by jorisvandenbossche** [comment]
Thanks for the feedback!

**Pull request merged**